oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleConstants.java

```diff
@@ -36,4 +36,28 @@ private OracleConstants() {
   public static final String NAME_DATABASE = "database";
   public static final String TNS_CONNECTION_TYPE = "TNS";
   public static final String TRANSACTION_ISOLATION_LEVEL = "transactionIsolationLevel";
+
+  /**
+   * Returns the Connection String for the given ConnectionType.
+   *
+   * @param connectionType SID/Service/SID
+   * @param host Host name of the oracle server
+   * @param port Port of the oracle server
+   * @param database Database to connect to
+   * @return Connection String based on the given ConnectionType
+   */
+  public static String getConnectionString(String connectionType,
+                                           String host,
+                                           int port,
```

Comment:
Are `host` and `port` nullable?

Reply:
Good catch, they can be null in some cases.